backend/packages/Upgrade/src/api/controllers/validators/MarkExperimentValidator.ts

```diff
@@ -14,4 +15,8 @@ export class MarkExperimentValidator {
   @IsNotEmpty()
   @IsDefined()
   public condition: string | null;
+
+  @IsNotEmpty()
+  @IsDefined()
+  public status: MARKED_DECISION_POINT_STATUS;
```

Comment:
Isn't this going to require clients to supply a value?  That's contrary to the strategy outlined in https://github.com/CarnegieLearningWeb/UpGrade/issues/275#issuecomment-1122783575 -- since Mathia won't immediately start using the new Java client library (indeed, we don't even _have_ a new Java client library yet! :) ), it will be a while before most API calls include this parameter.

Reply:
I was thinking just in terms js Client and made it a required parameter. I will make it optional until we update Java client.